test/runtime/monitor.go

```diff
@@ -56,26 +68,26 @@ var _ = Describe("RuntimeMonitorTest", func() {
 	})
 
 	AfterEach(func() {
-		docker.SampleContainersActions("delete", networkName)
+		docker.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
 	})
 
 	It("Cilium monitor verbose mode", func() {
 
-		res := cilium.Exec("config Debug=true DropNotification=true TraceNotification=true")
+		res := cilium.Exec("config MonitorDebug=true MonitorDropNotification=true MonitorTraceNotification=true")
```

Comment:
This is incorrect; fix with `Sprintf`  and calls to the monitor configuration constants.